atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/schema/stream/StreamTest.java

```diff
@@ -285,29 +285,28 @@ private void assertStreamHasBytes(InputStream stream, byte[] bytes) throws IOExc
         stream.close();
     }
 
-    @Test(expected = IllegalStateException.class)
-    public void readFromStreamWhenTransactionOpenThrowsException() throws IOException {
-        readFromGivenStreamWhenTransactionOpenThrowsException(defaultStore);
+    @Test()
+    public void readFromStreamWhenTransactionOpen() throws IOException {
+        readFromGivenStreamWhenTransactionOpen(defaultStore);
     }
 
-    @Test(expected = IllegalStateException.class)
-    public void readFromCompressedStreamWhenTransactionOpenThrowsException() throws IOException {
-        readFromGivenStreamWhenTransactionOpenThrowsException(compressedStore);
+    @Test()
+    public void readFromCompressedStreamWhenTransactionOpen() throws IOException {
+        readFromGivenStreamWhenTransactionOpen(compressedStore);
     }
 
-    private void readFromGivenStreamWhenTransactionOpenThrowsException(PersistentStreamStore store) {
+    private void readFromGivenStreamWhenTransactionOpen(PersistentStreamStore store) throws IOException {
```

Comment:
Do we also check that we can read correctly when the transaction is closed? I guess the other stream tests do this.